packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -215,6 +219,14 @@ class FlutterDevice {
         }
         return;
       }
+      try {
+        await service.streamListen(EventStreams.kExtension);
+      } on Exception catch (exception) {
```

Comment:
I don't think this is the right approach. If the structured errors are enabled and we can't listen to this stream, then a user would get no error output. I would say let it crash